types/pacote/index.d.ts

```diff
@@ -0,0 +1,162 @@
+// Type definitions for pacote 9.5
+// Project: https://github.com/npm/pacote#readme
+// Definitions by: Joel Spadin <https://github.com/ChaosinaCan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="node" />
+
+import npmFetch = require('npm-registry-fetch');
+
+export import extract = require('./extract');
+export import manifest = require('./manifest');
+export import packument = require('./packument');
+export import prefetch = require('./prefetch');
+export import tarball = require('./tarball');
+import { Readable } from 'stream';
+
+/**
+ * This utility function can be used to force pacote to release its references
+ * to any memoized data in its various internal caches. It might help free some
+ * memory.
+ */
+export function clearMemoized(): void;
+
+export interface Manifest {
+    name: string;
+    version: string;
+    dependencies: Record<string, string>;
+    optionalDependencies: Record<string, string>;
+    devDependencies: Record<string, string>;
+    peerDependencies: Record<string, string>;
+    bundleDependencies: false | string[];
+    bin: Record<string, string> | null;
+    _resolved: string;
+    _integrity: string;
+    _shrinkwrap: Record<string, unknown> | null;
+
+    engines?: Record<string, string>;
+    cpu?: string[];
+    os?: string[];
+    _id?: string;
+
+    [key: string]: unknown;
+}
+
+export interface PackageDist {
+    shasum: string;
+    tarball: string;
+    integrity?: string;
+    fileCount?: number;
+    unpackedSize?: number;
+}
+
+export interface Human {
+    name: string;
+    email?: string;
+    url?: string;
+}
+
+export interface PackageVersion {
+    name: string;
+    version: string;
+    dependencies?: Record<string, string>;
+    optionalDependencies?: Record<string, string>;
+    devDependencies?: Record<string, string>;
+    peerDependencies?: Record<string, string>;
+    directories: {};
```

Comment:
[npm's documentation for this field](https://github.com/npm/registry/blob/master/docs/REGISTRY-API.md#version) literally says "???". 🤷‍♀️

In every request I've tried, it's always shown up as an empty object.